plugins/gcp-projects/src/components/ProjectListPage/ProjectListPage.tsx

```diff
@@ -39,11 +39,17 @@ import { Link as RouterLink } from 'react-router-dom';
 
 const LongText = ({ text, max }: { text: string; max: number }) => {
   if (text.length < max) {
-    return <span>{text}</span>;
+    return (
+      <Typography color="primary" component="span">
```

Comment:
Probably missed it in a copy paste operation, will remove.

Reply:
Actually it does, this `LongText` is used only inside a link, I think if I don't put the primary color it will be overridden with the default typography color, which I think is black by default when using light theme.

Reply:
In same file around line `:82` it's being used inside the `renderLink` component, where it's already wrapped by a `Typography` with color primary, in case you want to check it.